README.md

```diff
@@ -66,7 +66,7 @@ If you prefer to have the application in English or any other language, you can
 
 ### Development
 
-1. Install node.js (version 10 required)
+1. Install node.js (refer to [.nvmrc](./.nvmrc) for required version)
```

Comment:
@McGiverGim what do you think? A bit worse ergonomics, but it won't be stale with version bumps.

Reply:
https://github.com/betaflight/betaflight-configurator/pull/2685

Reply:
Just awaiting this change to get it updated.